data_models/contents.py

```diff
@@ -7,24 +7,24 @@
 class Contents:
     """This class creates content_list which is list of extracted contents"""
 
+    CONTENTS_DICT_LIST = []
+
     def __init__(self):
         self.content_list = list()
 
     def add_content(self, _content):
         """This method appends extracted content to content_list"""
-        self.content_list.append(_content)
-
-    def get_formatted_list(self):
-        """This method converts the python class object list
-         into dictionary list"""
 
-        return_list = []
+        self.content_list.append(_content)
+        self.__dict__['content_list'].remove(_content)
+        self.__dict__.update({'content_list': self.updated_dict(_content)})
 
-        for obj in self.content_list:
-            if hasattr(obj, '__dict__'):
-                return_list.append(obj.get_formatted_dict())
+    def updated_dict(self, _content):
+        """This method updates the list with
+        dictionary of _content object"""
 
-        return return_list
+        self.CONTENTS_DICT_LIST.append(_content.__dict__)
+        return self.CONTENTS_DICT_LIST
```

Comment:
```
def add_content(self, _content):
        self.content_list.append(_content)
        self.__dict__.['content_list'].append(_content.__dict__)
```

Reply:
Do we need `CONTENTS_DICT_LIST`?

Reply:
Updates in next iteration. Thanks for the suggestion!